samples/test_relocation/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.8.2)
```

Comment:
Move this sample under samples/application_development

Also, do not call it test_***, if it is a test, it should not be under samples/